windows-builder: failed to build builder

## Expected Behavior

builder is biult.

## Actual Behavior

Failed to build builder

```
$ gcloud builds submit --config=builder/cloudbuild.yaml builder/
...
Step 4/7 : RUN GO111MODULE=on CGO_ENABLED=0 go build -o /go/bin/main
 ---> Running in c83d4a24c80a
go: downloading cloud.google.com/go v0.53.0
go: downloading github.com/masterzen/winrm v0.0.0-20190308153735-1d17eaf15943
go: downloading cloud.google.com/go/storage v1.5.0
go: downloading github.com/packer-community/winrmcp v0.0.0-20180921211025-c76d91
c1e7db
go: downloading github.com/pborman/uuid v1.2.0
go: downloading golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
go: downloading google.golang.org/api v0.19.0
go: downloading github.com/Azure/go-ntlmssp v0.0.0-20180810175552-4a21cbd618b4
go: downloading github.com/ChrisTrenkamp/goxpath v0.0.0-20170922090931-c385f95c6022
go: downloading github.com/gofrs/uuid v3.2.0+incompatible
go: downloading github.com/dylanmei/iso8601 v0.1.0
go: downloading github.com/nu7hatch/gouuid v0.0.0-20131221200532-179d4d0c4d8d
go: downloading github.com/google/uuid v1.0.0
go: downloading github.com/googleapis/gax-go/v2 v2.0.5
go: downloading google.golang.org/genproto v0.0.0-20200212174721-66ed5ce911ce
go: downloading golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550
go: downloading golang.org/x/net v0.0.0-20200202094626-16171245cfb2
go: downloading github.com/masterzen/simplexml v0.0.0-20160608183007-4572e39b1ab9
go: downloading google.golang.org/grpc v1.27.1
go: downloading go.opencensus.io v0.22.3
go: downloading golang.org/x/text v0.3.2
go: downloading github.com/golang/protobuf v1.3.3
go: downloading github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e
/go/pkg/mod/google.golang.org/grpc@v1.27.1/internal/channelz/types_linux.go:26:2: missing go.sum entry for module providing package golang.org/x/sys/unix (imported by google.golang.org/grpc/internal/channelz); to add:
        go get google.golang.org/grpc/internal/channelz@v1.27.1

The command '/bin/sh -c GO111MODULE=on CGO_ENABLED=0 go build -o /go/bin/main' returned a non-zero code: 1
ERROR
ERROR: build step 0 "gcr.io/cloud-builders/docker" failed: step exited with non-zero status: 1
```


## Steps to Reproduce the Problem

1. git clone https://github.com/GoogleCloudPlatform/cloud-builders-community
2. cd cloud-builders-community/windows-builder
3. follow https://github.com/GoogleCloudPlatform/cloud-builders-community/tree/master/windows-builder#getting-started

## Additional Info

need to update go.sum by running `go get google.golang.org/grpc/internal/channelz@v1.27.1`?

--- a/windows-builder/builder/go.mod
+++ b/windows-builder/builder/go.mod
@@ -13,4 +13,5 @@ require (
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
 	google.golang.org/api v0.19.0
 	google.golang.org/genproto v0.0.0-20200212174721-66ed5ce911ce
+	google.golang.org/grpc v1.27.1 // indirect
 )

